packer/elk.json

```diff
@@ -0,0 +1,51 @@
+{
+  "variables": {
+      "account_id": "753338109777",
+      "s3_bucket": "workflow-amis",
+      "build_number": "DEV",
+      "build_name": null,
+      "account_numbers": "",
+      "build_branch": "DEV"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "region": "eu-west-1",
+      "source_ami": "ami-274ecc50",
+      "instance_type": "m3.medium",
+      "ssh_username": "ubuntu",
+      "run_tags": {"Stage":"INFRA", "Stack":"packer", "App": "{{user `build_name`}}"},
+      "ami_name": "elk-stack_{{user `build_number`}}_{{isotime \"2006-01-02\"}}",
+      "ami_description": "AMI for ELK stack built by TeamCity: {{user `build_name`}}#{{user `build_number`}}",
+      "ami_users": "{{user `account_numbers`}}",
+      "tags": {"Name": "{{user `build_name`}}", "Build":"{{user `build_number`}}", "Branch":"{{user `build_branch`}}"}
+    }
+  ],
+
+  "provisioners" : [
+    { "type": "shell",
+      "inline": [
+        "#!/bin/sh -x",
+        "sudo add-apt-repository \"deb http://us.archive.ubuntu.com/ubuntu/ trusty universe multiverse\"",
+        "sudo add-apt-repository \"deb http://us.archive.ubuntu.com/ubuntu/ trusty-updates universe multiverse\"",
+        "sudo add-apt-repository \"deb http://us.archive.ubuntu.com/ubuntu/ trusty main restricted\"",
+        "sudo add-apt-repository \"deb http://packages.elasticsearch.org/logstash/1.4/debian stable main\"",
+        "sudo add-apt-repository \"deb http://packages.elasticsearch.org/elasticsearch/1.3/debian stable main\"",
+        "sudo add-apt-repository -y ppa:chris-lea/node.js",
+        "echo \"wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | apt-key add -\" | sudo sh",
```

Comment:
I think the key should be retrieved on https.
https://packages.elasticsearch.org/GPG-KEY-elasticsearch seems to work